### PR TITLE
Adds Medical and Prescription Medical/Security HUDS to the Loadout menu

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
Does what it says on the tin, adds the Medical Hud and Prescription Medical Hud/Security Hud to the loadout menu.

## Why It's Good For The Game
These two jobs kind of require these and the game saying **hee hoo blind boy** is only funny the first time around.

## Changelog
:cl:
add: You can now pick the Medical, Prescription Medical, and Prescription Security HUDS in the loadout menu.
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/client/loadout/_medical.dm [hunk 1]
```diff
@@ -5,3 +5,17 @@
 	path = /obj/item/clothing/mask/surgical
 	restricted_roles = MED_ROLES
 	restricted_desc = "Medical, including Brig Physician"
+
+/datum/gear/medhud
+	name = "Medical Hud"
+	category = SLOT_GLASSES
+	path = /obj/item/clothing/glasses/hud/health
+	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Geneticist", "Chemist", "Virologist")
```
Comment (CameronWoof) [inline modular_skyrat/code/modules/client/loadout/_medical.dm:13]: Paramedic? Brig Physician?

**Comment by Owai-Seek** [comment]
I'd rather have the ability to craft medical/security huds together with prescription glasses, instead of giving science one less thing to research.

I don't know all that much about security, but I know you can definitely do medical without a HUD, even if it's slightly more annoying.

**Comment by ShadeAware** [comment]
> 
> 
> I'd rather have the ability to craft medical/security huds together with prescription glasses, instead of giving science one less thing to research.
> 
> I don't know all that much about security, but I know you can definitely do medical without a HUD, even if it's slightly more annoying.
> 
> It also gives people reason to just, start as medical, then get unassigned, and yay, free gear.

1. Its not giving them anything less to research, because you're still going to have people who either lose theirs or they get destroyed. Thats like saying Security shouldn't have HUDGlasses at all roundstart because they _might_ get them at some point if Science exists (which they usually don't).

2. Quality of life is never a bad thing, and triage can mean life or death.

3. Strawman, literally nobody does this. If they do, Ahelp it. You can already get the HUDS that start in the lockers and desks around Medical too, so this is pretty much a moot point.

**Comment by CameronWoof** [comment]
Why not Brig Physician? They have access to medical storage.

**Pull request merged**